analytics/management/commands/populate_analytics_db.py

```diff
@@ -115,24 +106,16 @@ def handle(self, *args: Any, **options: Any) -> None:
             force_date_joined=installation_time,
         )
 
-        administrators_user_group = NamedUserGroup.objects.get(
-            name=SystemGroups.ADMINISTRATORS, realm=realm, is_system_group=True
-        )
-        can_administer_channel_group_default_name = Stream.stream_permission_group_settings[
-            "can_administer_channel_group"
-        ].default_group_name
-        can_administer_channel_group_default_value = NamedUserGroup.objects.get(
-            name=can_administer_channel_group_default_name,
-            realm=realm,
-            is_system_group=True,
-        )
         stream = Stream.objects.create(
             name="all",
             realm=realm,
             date_created=installation_time,
-            # TODO: Use default group instead of a fixed value.
-            can_remove_subscribers_group=administrators_user_group,
-            can_administer_channel_group=can_administer_channel_group_default_value,
+            can_remove_subscribers_group=get_stream_permission_default_group(
+                "can_remove_subscribers_group", realm
+            ),
+            can_administer_channel_group=get_stream_permission_default_group(
+                "can_administer_channel_group", realm
+            ),
```

Comment:
This might be a block of code that should be converted to use `do_create_realm` or similar helpers. Feels like a lot of basically duplicated logic here. Would be a nice way to not have to touch this file (and thus force everyone to re-build their database the next time they `provision`) every time we add a new one of these settings.

Fine as a follow-up, this is definitely better than what we had before.